std/numeric.d

```diff
@@ -1647,7 +1647,9 @@ unittest
 }
 
 /**
-Computes accurate sum of binary logarithms of input range $(D r).
+Computes sum of binary logarithms of input range $(D r).
+A worst-case error of roughly O(εN), average O(εN^^0.5).
+An absolute error of roughly is much less then naive sum of log2s has.
  */
```

Comment:
Compute the sum of binary logarithms of the input range r.
The error of this method is much smaller than with a naive sum of log2.
It is O(EN^^0.5) on average and O(EN) in the worst-case as opposed to O(...) for the naive sum.